base/java-tools/src/com/netscape/cmstools/ca/CACertRequestReviewCLI.java

```diff
@@ -99,22 +113,29 @@ public void execute(CommandLine cmd) throws Exception {
 
             System.out.println();
             System.out.println("Please review the certificate request in " + filename + ".");
-            System.out.println("Update the file if necessary, then select an action below.");
-            System.out.println();
 
-            while (true) {
-                System.out.print("Action (" + StringUtils.join(actions, "/") + "): ");
-                System.out.flush();
+            if (cmd.hasOption("file")) { // old process
 
-                BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
-                action = reader.readLine().trim().toLowerCase();
+                System.out.println("Update the file if necessary, then select an action below.");
+                System.out.println();
 
-                if (actions.contains(action)) break;
-            }
+                while (true) {
+                    System.out.print("Action (" + StringUtils.join(actions, "/") + "): ");
+                    System.out.flush();
+
+                    BufferedReader reader = new BufferedReader(new InputStreamReader(System.in));
+                    action = reader.readLine().trim().toLowerCase();
 
-            logger.info("Loading certificate request from " + filename);
-            String xml = new String(Files.readAllBytes(Paths.get(filename)));
-            reviewInfo = CertReviewResponse.fromXML(xml);
+                    if (actions.contains(action)) break;
+                }
+
+            } else { // new process
+
+                System.out.println("Update the file if necessary, then execute:");
+                System.out.println("  $ pki <authentication> ca-cert-request-<action> " + requestId + " --input-file " + filename);
```

Comment:
Idea. Can we detect this so the first half is at least copy-pasteable? So like, if you called:

```
$ pki -u username -P password ....
```

You'd get that all printed here?

Reply:
I think we can. Let me update the patch.

Reply:
This is actually more difficult than expected since the original objects are not available, so I just put ellipsis there for now.